lmdeploy/lite/quantization/awq.py

```diff
@@ -153,8 +160,12 @@ def smooth_ln_fcs(ln: torch.nn.Module,
     concat_w = torch.cat([fc.weight for fc in fcs], dim=0)
     w_scales = get_weight_scale(concat_w, group_size)
 
+    w_scales_pow = w_scales.pow(1 - alpha)
+    if w_scales_pow.min().item() == 0:
```

Comment:
w_scales are clamped.
It's impossible that `w_scales.pow(1-alpha) == 0`

Reply:
Mathematically, it is impossible, but from previous experience, it is possible.